Lib/gftools/scripts/gen_spac.py

```diff
@@ -27,40 +28,63 @@ def add_space_axis(font, amount):
         glyph_variations = gvar.variations.get(glyph_name)
         if not glyph_variations:
             continue
-        coords = [None] * len(glyph_variations[0].coordinates)
-        coords[0] = (amount, 0)
-        coords[-3] = (amount * 2, 0)
-        for idx in glyph.endPtsOfContours[:-1]:
-            coords[idx + 1] = (amount, 0)
-        tp = TupleVariation({"SPAC": (0.0, 1.0, 1.0)}, coords)
-        gvar.variations[glyph_name].append(tp)
+        min_coords = [None] * len(glyph_variations[0].coordinates)
+        min_coords[-3] = (min_amount, 0)
+        min_coords[-4] = (-min_amount, 0)
+        min_tp = TupleVariation({"SPAC": (-1.0, -1.0, 0.0)}, min_coords)
+        gvar.variations[glyph_name].append(min_tp)
+
+        max_coords = [None] * len(glyph_variations[0].coordinates)
+        max_coords[-3] = (max_amount, 0)
+        max_coords[-4] = (-max_amount, 0)
+        max_tp = TupleVariation({"SPAC": (0.0, 1.0, 1.0)}, max_coords)
+        gvar.variations[glyph_name].append(max_tp)
 
     name_table = font["name"]
 
     axis = Axis()
     axis.axisTag = "SPAC"
     axis.axisNameID = name_table.addMultilingualName({"en": "Spacing"})
-    axis.minValue = 0.0
+    axis.minValue = min_amount
     axis.defaultValue = 0.0
-    axis.maxValue = amount
+    axis.maxValue = max_amount
 
     fvar = font["fvar"]
     fvar.axes.append(axis)
     for inst in fvar.instances:
         inst.coordinates["SPAC"] = 0.0
     add_HVAR(font)
 
+    print("Update various VarStores")
+    stores = []
+    for table in ("MVAR", "HVAR", "BASE", "VVAR", "COLR", "GDEF"):
+        if table in font and hasattr(font[table].table, "VarStore"):
+            stores.append(font[table].table.VarStore)
+    nullRegion = otTables.VarRegionAxis()
+    nullRegion.StartCoord = -1
+    nullRegion.PeakCoord = 0
+    nullRegion.EndCoord = 1
+    for store in stores:
+        store.VarRegionList.RegionAxisCount = len(fvar.axes)
+        for region in store.VarRegionList.Region:
+            while len(region.VarRegionAxis) < len(fvar.axes):
+                region.VarRegionAxis.append(nullRegion)
+
 
 def main(args=None):
     parser = argparse.ArgumentParser(description="Add a SPAC axis to a variable font.")
     parser.add_argument(
         "font", type=TTFont, help="Path to the variable font file to modify."
     )
     parser.add_argument(
-        "--amount",
+        "min",
+        type=int,
+        help="Min amount of spacing to add",
```

Comment:
Maybe add asserts that this numbers is <=0, and max number is >=0? It's confusing whether one would use `--min=100` or `--min=-100`.